public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Cancun-Deneb (Dencun) FAQ
+description: Frequently asked questions regarding the Cancun-Deneb (Dencun) network upgrade
+lang: en
+---
+
+# Cancun-Deneb (Dencun) {#dencun}
+
+Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing data **blobs** for cheaper layer 2 (L2) rollup storage.
```

Comment:
```suggestion
Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing data **blobs** for cheaper [layer 2 (L2)](/layer-2/) rollup storage.
```

Reply:
I have a link to this a bit further down to avoid taking users away during the intro

Reply:
ok then maybe use glossary here?

Reply:
Sounds good, just added that in